### DISCUSSION TITLE
Any dk enjoyers?

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#5867] [General]
Some say dk would be op but I honestly don't think so https://youtu.be/3sE5SZq-btw
This is a prepatch run in sunwell and dk isnt even the best melee. They are good sure but they aren't even the best melee dps let alone the best dps.
They won't have sets unless you want to add them but that is a different convo.
I don't think they will be as strong as people say they are. Unless gargoyle is bugged in some way but that is a bug that will get fixed.
If you wanna use the argument that they would be op in pvp
1. PvP is dead, no other way of putting it that is just how is. 
2. Again unless gargoyle is bugged I really dont think so, sure they might be strong in 1v1 but not unkillable. I've seen plenty of ferals beat dk without los and I think feral historically has been bad vs dk
3. But what if they have a healer? Then cc or kill the healer. No different than if an arms war has a pocket healer
Unh can't even get lichbourne + gargoyle until 71

**Comment by balleny** [comment]
> Some say dk would be op but I honestly don't think so https://youtu.be/3sE5SZq-btw This is a prepatch run in sunwell and dk isnt even the best melee. They are good sure but they aren't even the best melee dps let alone the best dps. They won't have sets unless you want to add them but that is a different convo. I don't think they will be as strong as people say they are. Unless gargoyle is bugged in some way but that is a bug that will get fixed. If you wanna use the argument that they would be op in pvp

I would highly assume that the dks were also overall less well geared compared to the other chars that ran dozen of t6 raids before

> 1. PvP is dead, no other way of putting it that is just how is.
dks would likely add quite some disbalance to pvp and pve, we cannot be sure and would likely only find out if we ran an A/B test.

but, if you introduce the DK, there isnt really a way to go back.

> 2. Again unless gargoyle is bugged I really dont think so, sure they might be strong in 1v1 but not unkillable. I've seen plenty of ferals beat dk without los and I think feral historically has been bad vs dk

from my experience I would say ferals have quite an easy time against dks overall. I would see a lot more imbalance w.r.t. dks vs casters.

**Reply by Never6** [reply]
If it was a gear difference on ptr I would be happy to test dks on ptr vs other classes, but as of right now you cannot even created a dk on ptr

Even if I'm the only person who wouldn't mind dk on live server. I think we can agree dk on ptr would only be a good thing.
DK could be excuse my French bugged as fucking shit rn and nobody would know until 74 which is who knows how far away

**Reply by heyitsbench** [reply]
> I think we can agree dk on ptr would only be a good thing.

By the way, I've made a suggestion to open up PTR DK [here](https://github.com/chromiecraft/chromiecraft/discussions/5948).

**Comment by Vyperhand** [comment]
I agree.  We're transitioning towards custom, and CC is really the only game in town for WotLK PvE.  At this point, I can't see any harm in releasing them early, and can see significant upsides in attracting another segment of player base by offering the unique experience of raiding TBC content while relevant with some DKs in the mix.

**Comment by Never6** [comment]
![sadge](https://github.com/chromiecraft/chromiecraft/assets/66432845/5d8966f8-0042-4cc7-a4ec-9b1b4f5071ca)
